src/database/models/bookmark.js

```diff
@@ -0,0 +1,21 @@
+import mongoose from 'mongoose';
+const { model, Schema } = mongoose;
+
+const bookMarkSchema = new Schema({
+    tag_id: {
+        type: Number,
```

Comment:
Let's change `Number` to `Schema.Types.ObjectId`

Reply:
I've changed it.